src/DssSpell.sol

```diff
@@ -56,163 +49,189 @@ contract DssSpellAction is DssAction {
-        // Forum: https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/9
-
-        // Launch Project - 4,500,000.00 DAI - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
-        DssExecLib.sendPaymentFromSurplusBuffer(LAUNCH_PROJECT_FUNDING, 4_500_000);
-        // Launch Project - 820.00 MKR - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
-        MKR.transfer(LAUNCH_PROJECT_FUNDING, 820.00 ether); // NOTE: ether is a keyword helper, only MKR is transferred here
-
-        // ---------- Update doc parameter ----------
-        // Forum: https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/14
-
-        // Update HVBank (RWA009-A) doc to QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT
-        _updateDoc("RWA009-A", "QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT");
-
-        // ---------- Spark D3M line increase ----------
-        // Forum: https://forum.makerdao.com/t/spark-spell-proposed-changes/23298
-        // Poll: https://vote.makerdao.com/polling/QmdQSuAc
-
-        // Increase the line by 400 million from 800 million to 1.2 billion Dai
-        DssExecLib.setIlkAutoLineDebtCeiling("DIRECT-SPARK-DAI", 1200 * MILLION);
+        // Set Liquidation Ratio to 10,000%.
+        // Note: We are using low level methods because DssExecLib only allows setting `mat < 1000%`: https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L717
+        DssExecLib.setValue(MCD_SPOT, "RETH-A", "mat", 100 * RAY);
+
+        // Note: Update collateral price to propagate the changes
+        DssExecLib.updateCollateralPrice("RETH-A");
+
+        // ---------- SBE parameter changes ----------
+        // Forum: https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441
+
+        // Increase bump by 20,000, from 30,000 to 50,000 DAI
+        DssExecLib.setValue(MCD_VOW, "bump", 50 * THOUSAND * RAD);
+
+        // Increase hop by 10,512, from 15,768 to 26,280 Seconds
+        DssExecLib.setValue(MCD_FLAP, "hop", 26_280);
+
+        // ---------- Approve HV Bank (RWA009-A) DAO Resolution ----------
+        // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/21
+
+        // Mainnet - Add RWA009_A_INPUT_CONDUIT_URN_USDC deployed at 0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1 to the chainlog
+        // Forum: https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/15
+        DssExecLib.setChangelogAddress("RWA009_A_INPUT_CONDUIT_URN_USDC", RWA009_A_INPUT_CONDUIT_URN_USDC);
+
+        // Call <conduit>.rely(MCD_ESM) to allow ESM module to deny the pause proxy in SwapInputConduit contracts
+        DssExecLib.authorize(RWA009_A_INPUT_CONDUIT_URN_USDC, MCD_ESM);
+
+        // Goerli - Add RWA009_A_INPUT_CONDUIT_URN_USDC deployed at 0xddd021b7e3Bfbad19c7D455EB7976DCe51180141  to the chainlog
```

Comment:
Cosmetic comment: shouldn't we remove these, as they are confusing (Goerli comments, followed by notes about mainnet, the one below is actually being executed on mainnet)

Reply:
There is no rule about it, but my current approach is to transfer all instructions into the spell in order to not forget/miss anything. It looks kind of dumb in regards to goerli-prefixed instructions, but is still very explicit. I'm not fixed on this, but:
- It's more explanatory than a skipped/missing instruction, especially for non-technical reviewers
- You as a reviewer have a place to leave a comment and disagree with the explanation

Reply:
Honestly, I don't remember seeing this in previous spells.

It feels kind of weird to have Goerli references in a Mainnet spell. The other way around happens, specially when it comes to payments.

In the past, the Goerli instruction for such authorization would be in the exec sheet, but not in the exec copy, because the copy is meant to be used in Mainnet only.

I believe we should only change this pattern if we all agree to add a specific rule to it in the crafter checklist.

Reply:
Agree, after searching for an archive example, could only find `ignore in goerli` leftovers. So I've removed goerli-related instructions from the spell via 1fc4b16